invoices/invoice_expiry_watcher_test.go

```diff
@@ -203,3 +204,114 @@ func TestInvoiceExpiryWhenAddingMultipleInvoices(t *testing.T) {
 	test.watcher.Stop()
 	test.checkExpectations()
 }
+
+// TestExpiredHodlInv tests expiration of an already-expired hodl invoice
+// which has no htlcs.
+func TestExpiredHodlInv(t *testing.T) {
+	t.Parallel()
+
+	creationDate := testTime.Add(time.Hour * -24)
+	expiry := time.Hour
+
+	test := setupHodlExpiry(
+		t, creationDate, expiry, 0, channeldb.ContractOpen, nil,
+	)
+
+	test.assertCanceled(t, test.hash)
+	test.watcher.Stop()
+}
+
+// TestAcceptedHodlNotExpired tests that hodl invoices which are in an accepted
+// state are not expired once their time-based expiry elapses.
```

Comment:
Looks like the description is a bit off.

Reply:
I think it is? `setupHodlExpiry` adds an accepted invoice with now+1h expiry, and then we tick the test clock to that time. I've added another invoice as a control to check that we're actually expiry invoices at now+1h, updated the comment to reflect that 👍